packages/react-location/src/index.tsx

```diff
@@ -689,6 +697,14 @@ export class RouterInstance<
       const age = Date.now() - (match.updatedAt ?? 0)
 
       if (!match.maxAge || age > match.maxAge) {
+        if (match.route.unloader) {
+          if (match.status === 'resolved') {
+            match.route.unloader(match, match.ownData)
+          } else {
+            match.route.unloader(match)
+          }
+        }
+
```

Comment:
I'm not the best with the lifecycle of promises in JS. This will fire-and-forget the unloader promise, without awaiting on it or anything. Is this ok?